tests/specs/benchmarks/verification.k

```diff
@@ -148,6 +148,17 @@ module VERIFICATION-HASKELL [symbolic, kore]
 
     rule maxUInt160 &Int Y => Y requires 0 <=Int Y andBool Y <=Int maxUInt160 [simplification]
 
+    rule ( #sizeByteArray(#ecrec( _, _, _, _ )) <=Int 32 ) => true  [simplification]
+
+    rule MEM [ K1 := BUF1:ByteArray                 ] [ K2 := #ecrec(HASH, SIGV, SIGR, SIGS) ] 
+      => MEM [ K2 := #ecrec(HASH, SIGV, SIGR, SIGS) ] [ K1 := BUF1                           ] 
+        requires (32 <=Int K1 -Int K2) andBool (K2 <Int K1) [simplification]
+
+    rule ( maxUInt160 &Int #asWord ( padLeftBytes( #ecrec(HASH, SIGV, SIGR, SIGS), N, 0 ) ) ) => #asWord ( padLeftBytes( #ecrec(HASH, SIGV, SIGR, SIGS), N, 0 ) )
+      requires #asWord(#ecrec(HASH, SIGV, SIGR, SIGS)) <=Int maxUInt160 [simplification]
+
+    rule 0 <Int #asWord ( padLeftBytes ( #ecrec( HASH, SIGV, SIGR, SIGS ), _, 0 ) ) => notBool ( #ecrecEmpty ( HASH, SIGV, SIGR, SIGS ) )  [simplification]
```

Comment:
It's a little worrisome that we have to include lemmas about `padLeftBytes`, and indicates taht somewhere we are not stopping some symbol from being evaluated that should not be (with the `--concrete-rules` flag).

I've

Reply:
These have all changed, I believe, to simplifications about `#padToWidth`? @nwatson22